linkml_model_enrichment/bioportal-enum-annotation.py

```diff
@@ -8,17 +10,42 @@
 from strsimpy.cosine import Cosine
 import yaml
 import re
-# import os
 from yaml import SafeDumper
 import click
 
 
+# import os
+# import requests
+# from curieutil import CurieUtil
+# from prefixcommons import contract_uri
+
+# go_prefixes_url = 'https://raw.githubusercontent.com/prefixcommons/biocontext/master/registry/go_context.jsonld'
+# r = requests.get(go_prefixes_url)
+#
+# mapping = CurieUtil.parseContext(r.json())
+# curie = CurieUtil(mapping)
+
+# WRITES OUTPUT TO STDOUT
+
+# 2021-04-14
```

Comment:
minor: some of this belongs in git commit messages

in general no need to retain commented out code, git takes care of history

in general most docs should go in docstrings so they are visible at IDE/RTD level